x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/utils.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const getDateString = (offSetDays?: number) => {
```

Comment:
FYI this util comes from the standalone UI, I basically just copied it over as-is, def. open to changes if we want them